core/unit_test/TestBitManipulationBuiltins.hpp

```diff
@@ -828,7 +828,7 @@ struct TestBitCastFunction {
     }
 
 #if defined(KOKKOS_ENABLE_CUDA) && \
-    defined(KOKKOS_COMPILER_NVHPC)  // FIXME_NVHPC
+    defined(KOKKOS_COMPILER_NVHPC)  // FIXME_NVHPC 23.7
```

Comment:
Why did you add 23.7 here?

Reply:
Book keeping to clarify that yes during this review I checked that compiler. Next time we go through all of this we can update.

Reply:
To make that clear: I explicitly tried taking that ifdef out and checked that it still fails.

Reply:
Fine